### DISCUSSION TITLE
save actions button

### DISCUSSION THREAD

**Discussion by soluzione-software** [Laravel-Backpack/community-forum#442] [Q&A (Help)] [answered]
I have backpack 5.5 with laravel 9
when i try to select a different save action (for sample save_and_new or save_and_preview ecc..) it seems that not work.
when i clicked the label on context menù that appear, the label of the button not changed and when the button is pressed it do save_and_back (default)
![image](https://user-images.githubusercontent.com/49692407/225883286-983adb52-300b-4111-a239-87d268494217.png)

why it not work?
thanks

**Comment by promatik** [comment]
Hi @soluzione-software!

I'm not sure if I understood your issue ...
Can you please go to our demo page: https://demo.backpackforlaravel.com/admin/article and try it there?

I just tested it, and it works as expected.
Is the behaviour there expected? is it different from yours?

**Comment by soluzione-software** [comment, marked as answer]
thanks promatik. it was a previous javascript problem, i see that the action works with jquery. I solve.
